### PR TITLE
Send the usage statistics to Google Analytics

### LINKED ISSUE #1286: Add Google Measurement Protocol into CLI
To improve user experience we would like to collect anonymous usage data using [Google Measurement Protocol](https://developers.google.com/analytics/devguides/collection/protocol/v1).

After each CLI run we should submit an [event into Google Analytics](https://developers.google.com/analytics/devguides/collection/protocol/v1/devguide#event) with category `CLI`, action `Run`, and label being a JSON capturing command line arguments and exit code. Tracking id: `UA-106571369-3`

Anonymous User ID that is passed with the HTTP call needs to be auto-generated and persisted locally.

Usage reporting should be turned on by default, however, users should be able to opt-out from sending usage data by running `neuro config opt-out`.

Upon first run of the CLI we should display the message saying: `In order to improve user experience, we are anonymously tracking usage data. You can opt-out by running "neuro confing opt-out"`. The message should be displayed once after first installation of `neuro`.

### PR DESCRIPTION
**Pull request by asvetlov**
Fix #1286

### CODE REVIEW

**Comment by asvetlov** [comment]
The PR is tested manually, I'm working on adding some unit tests.

File: neuromation/cli/main.py [hunk 1]
```diff
@@ -196,6 +197,17 @@ def print_options(
         "together with option '--trace'. On by default."
     ),
 )
+@option(
+    "--skip-stats/--no-skip-token",
```
Comment (anayden) [inline neuromation/cli/main.py]: Looks like a typo: should be `--no-skip-stats`

Comment (asvetlov) [reply neuromation/cli/main.py]: You are right, thanks!

**Comment by asvetlov** [comment]
Tests are done I think

**Pull request merged**